### ISSUE TITLE
Add MDS in a box

### ISSUE THREAD

**Issue by davidgasquez** [davidgasquez/awesome-duckdb#13]
Hey @matsonj, what do you think could be a great section/place for [MDS in a box](https://github.com/matsonj/nba-monte-carlo)? Was thinking about `Projects Powered By DuckDB` but would love to check if you have other suggestions.

**Comment by matsonj** [comment]
Yeah, I think ```Projects Powered by DuckDB``` is fine. There is a lot of neat stuff out there like that. 

I would also add a link to this post under Media /blogs - https://duckdb.org/2022/10/12/modern-data-stack-in-a-box.html

**Comment by matsonj** [comment]
this article is great too: https://dagster.io/blog/duckdb-data-lake

**Comment by matsonj** [comment]
one more: https://medium.com/@rahul_39479/a-super-simple-and-highly-customizable-modern-data-stack-mds-in-a-box-using-dlt-and-motherduck-d6482ce9fab4

**Comment by davidgasquez** [comment]
Let's go!

https://github.com/davidgasquez/awesome-duckdb#projects-powered-by-duckdb
